From 9b9de7f2dbcf295becdcbbf16dbb9d74190e9936 Mon Sep 17 00:00:00 2001
From: Lars Wendler <polynomial-c@gentoo.org>
Date: Tue, 12 Jul 2016 15:46:58 +0200
Subject: [PATCH] Don't include internal zip library when USE_MINIZIP is
 defined.

According to CmakeLists.txt file USE_MINIZIP enables linking against system
minizip library but the code in src/util/vfs/vfs-zip.c doesn't reflect
that.

Signed-off-by: Lars Wendler <polynomial-c@gentoo.org>
---
 src/util/vfs/vfs-zip.c | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/util/vfs/vfs-zip.c b/src/util/vfs/vfs-zip.c
index 83936f6..e06ae7b 100644
--- a/src/util/vfs/vfs-zip.c
+++ b/src/util/vfs/vfs-zip.c
@@ -34,7 +34,11 @@ enum {
 	BLOCK_SIZE = 1024
 };
 #else
+#ifdef USE_MINIZIP
+#include <minizip/unzip.h>
+#else
 #include "third-party/zlib/contrib/minizip/unzip.h"
+#endif
 #include "util/memory.h"
 
 struct VDirEntryZip {
-- 
2.9.1

